ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/denyRequestFlow.test.ts

```diff
@@ -0,0 +1,84 @@
+
+const comment = 'Defer me';
+const expiry = 'When all CVEs are fixable';
+const scope = 'All images';
+
+describe('Exception Management Request Details Page', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+            deferAndVisitRequestDetails({
+                comment,
+                expiry,
+                scope,
+            });
+        }
+    });
+
+    after(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    it('should be able to deny a request if approval permissions are granted', () => {
+        cy.get('button:contains("Deny request")').click();
+        cy.get('div[role="dialog"]').should('exist');
+        getInputByLabel('Denial rationale').type('Denied');
+        cy.get('div[role="dialog"] button:contains("Deny")').click();
+        cy.get('div[role="dialog"]').should('not.exist');
+        cy.get('div[aria-label="Success Alert"]').should(
+            'contain',
+            'The vulnerability request was successfully denied.'
+        );
+    });
+
+    it('should be able to see how many CVEs will be affected by a denial', () => {
+        cy.get('table tbody tr:not(".pf-c-table__expandable-row")').then((rows) => {
+            const numCVEs = rows.length;
+            cy.get('button:contains("Deny request")').click();
+            cy.get('div[role="dialog"]').should('exist');
+            cy.get(`div:contains("CVE count: ${numCVEs}")`).should('exist');
+        });
+    });
+
+    it('should be able to see how many images will be affected by a denial', () => {
+        cy.get('table tbody tr:not(".pf-c-table__expandable-row")').then((rows) => {
+            let totalNumAffectedImages = 0;
+            rows.each((_, row) => {
+                const numAffectedImages = parseInt(
+                    Cypress.$(row).find('td[data-label="Affected images"]').text()
+                );
+                totalNumAffectedImages += numAffectedImages;
+            });
+            cy.get('button:contains("Deny request")').click();
+            cy.get('div[role="dialog"]').should('exist');
+            cy.get(`div:contains("Affected images: ${totalNumAffectedImages}")`).should('exist');
+        });
```

Comment:
I think this will work if there is only ever a single CVE in the request, but if there are multiple CVEs I don't think the logic will hold. If the list of CVEs has overlap in the affected images, the total from the table will be different from the total in the modal, right?

Reply:
Hmm yeah, I get your point. There can be an overlap. From a testing point of view, we can omit this test case. In terms of the UX, I'll need to talk to Mansur about what would be the best option here:
1. Removing the "Affected images" line
2. Keeping it but making sure it uses the correct value
3. Keeping it and listing the actual images out (might be more work)

Reply:
Looks like #2 and #3 might not be entirely possible as of right now. I mean we could try it, but it'll require quite a bit more effort. I'm double checking with Mansur if #1 is reasonable

Reply:
Ah I didn't realize we computed that value client side. In that case I'd vote for (1) as well, doing either of the other two would definitely be more work, and potentially put extra load on the server for something that isn't essential in the UI.